config/application.yml.default

```diff
@@ -418,6 +418,7 @@ test:
   phone_setups_per_ip_track_only_mode: 'false'
   piv_cac_verify_token_secret: 3ac13bfa23e22adae321194c083e783faf89469f6f85dcc0802b27475c94b5c3891b5657bd87d0c1ad65de459166440512f2311018db90d57b15d8ab6660748f
   poll_rate_for_verify_in_seconds: '1'
+  rack_timeout_service_timeout_seconds: 9_999_999_999
```

Comment:
I'm less sure about doing this in test, I'd rather not have a test wait 300 years to fail 😛

Reply:
😬 so what should we set it to?

Reply:
If we haven't seen any issues in test, leaving the default seems fine for now?

Reply:
I thought I was getting segfaults in test but I guess not! Will close this for now and we can always redo if others have issues